accProcess.py

```diff
@@ -203,6 +203,26 @@ def main():
                             metavar='True/False', default=False, type=str2bool,
                             help="""Save intensity distribution
                              (default : %(default)s)""")
+    parser.add_argument('--psd',
+                            metavar='True/False', default=False, type=str2bool,
+                            help="""Calculate power spectral density for 24 hour 
+                                    circadian period
+                             (default : %(default)s)""")
+    parser.add_argument('--fourierFrequency',
+                            metavar='True/False', default=False, type=str2bool,
+                            help="""Calculate dominant frequency of sleep
+                             (default : %(default)s)""")
+    parser.add_argument('--m10l5',
+                            metavar='True/False', default=False, type=str2bool,
+                            help="""Calculate relative amplitude of most and 
+                                    least active acceleration periods
+                             (default : %(default)s)""")
+    parser.add_argument('--fourierWithAcc',
```

Comment:
Make this code directly follow '--fourierFrequency' (i.e. --m10l5' should not break the flow in between them)

Reply:
resolved in bbe4536c65b673e2a6d7ec09277fc17e132c6837